pkg/deploy/generator/resources_rp.go

```diff
@@ -519,6 +519,8 @@ cat >/etc/td-agent-bit/td-agent-bit.conf <<'EOF'
 	Port 29230
 EOF
 
+export AZURE_CLOUD_NAME="AzureUSGovernment"
```

Comment:
This is not the right way to hardcode. Please remove from this PR as this is not the goal of this PR too.